app/main.py

```diff
@@ -1,3 +1,17 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Any:
+    cache_dict = {}
+
+    def wrapper(*args) -> Any:
+        if hash(args) in cache_dict:
```

Comment:
You don't need to use the hash() function, the dictionary will do it under the hood. Just store args as a key.